### DISCUSSION TITLE
Request:プレゼンテーション機能の最後の一枚に出てくる「おしまい」が場違いになることがあるので変更してほしい

### DISCUSSION THREAD

**Discussion by TatsuyaIse** [growilabs/growi#6490] [Help / Questions] [answered]
### 概要
- プレゼンテーション機能の最後のページに出てくる「おしまい」を無くすか、カスタマイズができるようにしてほしい

### 起因
GROWI の Slack ワークスペースの #help チャンネルに来た以下の要望から起票

> お世話になっております。プレゼンテーション機能について質問させていただきたいと思います。
社内外を問わず会議資料を GROWI ページで作成し、会議の場で「プレゼンテーション」機能を利用することがよくありますが、最後に表示される「おしまい」が場の雰囲気に合わないことで、プレゼンテーション機能を使えない場合があります。
現在使用中の GROWI は v5.1.2 (Docker Compose 版)ですが、この「おしまい」を非表示にするか、任意の文字列に変更する設定を追加するのは、今後の機能としていかがでしょうか？
ご検討いただけますと幸いです。

**Comment by TatsuyaIse** [comment]
custom scripts で対応できるとのこと、やり方はまだ不明なので検証してみます。
消したいのはココ → https://github.com/weseek/growi/blob/master/packages/app/src/server/views/page_presentation.html#L59

**Comment by yuki-takei** [comment, marked as answer]
「おしまい」ページは GROWI v6 以降で削除されました。
